README.md

```diff
@@ -16,97 +16,102 @@ When the reader has completed this Code Pattern, they will understand how to:
 
-### 3. Run the Application
-    
-   #### 3.1 Install the dependecies 
+## 3. Run the application
+
+To run the application, follow these steps:
+
+### 3.1 Install the dependecies 
    
-  - Goto the cloned repo from [step 1](#1-clone-the-repo), in Terminal run the following commands to install the required python libraries and run the app
-    - Install Required python libraries, by running the following command:
-    ```bash
-    $ pip install -r requirements.txt
-    ```
+Go to the cloned repo from [step 1](#1-clone-the-repo). In your terminal, run the following commands to install the required python libraries and run the app
     
-    - Run the application as follows:
-    ```bash
-    $ python app.py
-    ```
-    #### 3.2 Create a table and load it with required data
+    1. Install the required Python libraries by running the following command:
     
-    - Open the following URL in the browser:
+        ```bash
+        $ pip install -r requirements.txt
+        ```
     
-    `http://localhost:5000/create`
+    1. Run the application as follows:
     
-    - This will create `screen` and `userdetails` tables in the `Crunchy Data PostgreSQL` database  and load it with required data.
+        ```bash
+        $ python app.py
+        ```
+ 
+### 3.2 Create a table and load it with required data
     
-   > Note: Please be patient, this step will take a while
+Open the following URL in the browser: `http://localhost:5000/create`
     
-   #### 3.3 Open the booking application
+This will create `screen` and `userdetails` tables in the `Crunchy Data PostgreSQL` database  and load it with required data.
+    
+    > Note: Please be patient, this step will take a while
+    
+### 3.3 Open the booking application
+   
+Open the application by following the promptss or opening the `http://localhost:5000/`.
    
    ![](doc/source/images/create.png)
-   - Click on `here` hyperlink as shown above
-   - Alternative you can open `http://localhost:5000/`, to open the application
-   - You will now have the booking application on your screen as follows
    
+The booking application showing the seats in a theater should lead on your screen.
    ![](doc/source/images/booking.png)
    
     
-### 4. Analyse the results
+### 4. Analyze the results
 
-   - Type your `name`, `phone number` and `select the seats` as shown in the pictue below,
-   
-   ![](doc/source/images/reserveseats.gif)
+Now, let's play around with the app to see how it works.
+
+1. Select the seats you want, type your name and phone number in the fields and click **Reserve Seats**. 
+
+    ![](doc/source/images/reserveseats.gif)
    
-   - Your booking will be showcased in the booking details tab. Click on `Reserve More Tickets` to reserve more seats.
+1. Your booking will be showcased in the booking details tab <!--EM: I don't see this tab. DO you mean the Reserved tab?-->. Click on `Reserve More Tickets` to reserve more seats.
```

Comment:
As soon as you select the seats and book the tickets (Click on Reserve tickets). You will be redirected to a booking details tab. Please check https://raw.githubusercontent.com/IBM/movie_ticket_booking_application_using_crunchydb/master/doc/source/images/viewusers.gif.